README.md

```diff
@@ -1,221 +1,10 @@
 # Matrix-appservice-slack
-A Matrix &lt;--> Slack bridge
+A Matrix &lt;--&gt; Slack bridge
 
```

Comment:
We should have some badges here for things like the matrix room, docker image, etc :)